index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Nancy</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+    <script src='main.js'></script>
+    <link href='https://fonts.googleapis.com/css?family=Open Sans' rel='stylesheet'>
+    <script src="https://kit.fontawesome.com/1f371ca31b.js" crossorigin="anonymous"></script>
+</head>
+<body>
+
+ <!-- NAVIGATION HEADER -->
+
+    <header class="wrapper">
+        <nav>
+            <div class="flex-1 icon1">
+                <a href="#top-page"> 
+                 <img class="icon" src= "homepage.png" alt="homePage"/>
+                 home
+                </a>
+            </div>
+            <div class="flex-1 icon1">
+                <a href="#importantThings"> 
+                 <img class="icon" src="heart.png" alt="importantToMe">
+                 importantToMe
+                </a>
+            </div>
+            <div class="flex-1 icon2">
+                <a href="#topPurchases"> 
+                 <img class="icon" src="purchases.png" alt="topTenPurchases"/>
+                 top10Purchases
+                </a>
+            </div>
+            <div class="flex-1 icon3">
+                <a href="#FAQ"> 
+                 <img class="icon" src="questions.png" alt="FAQ"/>
+                 FAQ
+                </a>
+            </div>
+            <div class="flex-1 icon4">
+                <a href="#wouldYouRather"> 
+                 <img class="icon" src="choice.png" alt="wouldYouRather"/>
+                 wouldYouRather? 
+                </a>
+            </div>
+            <div class="flex-1 icon4">
+                <a href="#spotify"> 
+                 <img class="icon" src="spotify.png" alt="spotify"/>
+                 spotifyFaves 
+                </a>
+            </div>
+        </nav>
+    </header>
+
+ <!-- MAIN PAGE -->
+
+<section class="bottom-wrapper">
+
+    <!-- 'HOME' SECTION -->
+
+        <div class="flex-2" id="top-page">
```

Comment:
Bit more on semantic HTML, update main page to use main tag and then sections can use the section tag rather than a div. Screenreaders much prefer this too. Try out using dev tools to see the difference! 

```suggestion
        <section class="flex-2" id="top-page">
```